omnibus/resources/agent/msi/cal/CustomAction.cpp

```diff
@@ -368,6 +368,7 @@ extern "C" UINT __stdcall DoUninstall(MSIHANDLE hInstall) {
     return WcaFinalize(er);
 }
 
+BOOL DeleteFilesInDirectory(const wchar_t* dirname, const wchar_t* ext);
```

Comment:
Nit: since `DeleteFilesInDirectory` is defined in a different `cpp`, from a project standpoint I think it might make more sense to have this in one of the header files (or a new one), and include it, as opposed to a forward declaration here.

Reply:
it's not a nit... it was lazy.  Fixed.